ComfyUI/comfy_api_nodes/nodes_luma.py

```
@@ -715,6 +954,131 @@ def _convert_to_keyframes(
+                path=f"/proxy/luma/generations/{response_api.id}",
+                method=HttpMethod.GET,
+                request_model=EmptyRequest,
+                response_model=LumaGeneration,
+            ),
+            completed_statuses=[LumaState.completed],
+            failed_statuses=[LumaState.failed],
+            status_extractor=lambda x: x.state,
+            result_url_extractor=image_result_url_extractor,
+            node_id=unique_id,
+            progress_extractor=lambda x: 1.0 if getattr(x, "state", None) == LumaState.completed else 0.0,
+            auth_kwargs={"luma_api_key": api_key},
```

Ruff (0.12.2)

Undefined name `api_key`

(F821)